src/database/queries/AddressQueries.js

```diff
@@ -88,11 +88,49 @@ class AddressQueries {
 		}
 	}
 
+	static getToken(
```

Comment:
I need to review this getToken function more in-depth because token ownership gets very funky when dealing with an ERC-1155 token. Namely, a single "NFT"-type 1155 token can have more than one owner, so this function can return a lot of rows.